gammapy/cube/tests/test_sherpa_.py

```diff
@@ -1,10 +1,14 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 from numpy.testing.utils import assert_allclose
+import numpy as np
 from astropy.io import fits
+from astropy.coordinates import SkyCoord
 from ...datasets import gammapy_extra
+from ...irf import EnergyDispersion
 from ...utils.testing import requires_dependency, requires_data
 from .. import SkyCube
+from ..sherpa_ import NormGauss2DInt
```

Comment:
@JouvinLea I think this line causes the test fails on Travis-CI. The import should be moved to `testCombinedModel3DInt()` and `testCombinedModel3DIntConvolveEdisp`, as you did for the other imports.